application/app/src/main/java/com/example/buber/Services/ApplicationServiceHelper.java

```diff
@@ -0,0 +1,9 @@
+package com.example.buber.Services;
+
+import com.example.buber.Model.User;
+
+public interface ApplicationServiceHelper {
+
+
+    User aftersuccessfulCreataAccount(String DOCID);
```

Comment:
Typo in this and where this is referenced.

Reply:
ApplicationService.java extends ApplicationServiceHelper